src/lib/proof_cache_tag/lmdb/proof_cache_tag.ml

```diff
@@ -0,0 +1,44 @@
+open Async
+open Core
+module Cache = Disk_cache.Make (Pickles.Proof.Proofs_verified_2.Stable.Latest)
```

Comment:
```suggestion
module Cache = Disk_cache.Make (Mina_base.Proof)
```

Reply:
There is a problem with this change as Mina_base.Proof does not deriving bin_io

Reply:
@dkijania then `Mina_base.Proof.Stable.Latest`?